docs/contributing/01-start-here/05-development/02-windows-installation.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Windows Installation
+id: windows
+keywords: [Wing contributors, contributors, windows, installation]
+---
+
+## How to prepare for take-off on windows? 🐤
+
+We strongly recommend to set the tools up within WSL as some of the scripts
+don't support windows and expect the unix tooling.
+
+1. Install WSL 2: [💡 GUIDE](https://learn.microsoft.com/en-us/windows/wsl/install#install-wsl-command)
+    - Make sure to restart your machine after installation
+    - You can verify your wsl setup with `wsl -l -v`
+    - ⚠ All steps below should be done within WSL
+2. Install cc for rust: [💡 MORE INFO](https://stackoverflow.com/questions/52445961/how-do-i-fix-the-rust-error-linker-cc-not-found-for-debian-on-windows-10)
+3. Install [Rust]: [💡 GUIDE](https://rustup.rs/)
+    - Make sure to run `source "$HOME/.cargo/env"` as shown in the console at the end of installation (else cargo and rustc wont be found)
+4. Install [Node.js] v18: [💡 GUIDE](https://learn.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-wsl#install-nvm-nodejs-and-npm)
+    - `nvm install 18` should install the correct version of node
+  - You can verify your node setup with `node -v`
+5. Enable [PNPM] v8:
+    - For pnpm you can simply run `corepack enable`
+    - You can verify your pnpm setup with `pnpm -v`
+6. Install [Docker]:
+    - Needed to build the grammar as WASM for the web-based playground and to run unit tests
+    - WSL now comes with systemd, meaning you don't need docker desktop anymore
+    - Enable systemd [💡 GUIDE](https://devblogs.microsoft.com/commandline/systemd-support-is-now-available-in-wsl/#how-can-you-get-systemd-on-your-machine)
+    - Install docker [💡 GUIDE](https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository)
```

Comment:
hmmm not sure if we should maintain installation guides with OS specific steps. Since the development guide has some overlap. My main concern is guide sprawl, and the overhead that comes with updating these instructions. 

Maybe it would make sense to just add a section in the development page that explains to users that they will need to use WSL as for all the installation steps. 

Let me know what you think

Reply:
Yeah wasn't sure either. The information that WSL is needed is the most important one. Apart from that it is mostly FAQ kinda information for most common issues one might run into when installing it in WSL.